_installation/single-multiple.md

```diff
@@ -10,13 +10,20 @@ Babelfish offers two modes of operation:
 - `single-db`
 - `multi-db`
 
-What is the purpose of these two modes? In MS SQL, it is really cheap to perform
-joins between tables located in different databases. In PostgreSQL, it is also
-possible to join between databases, but it is far more expensive to do so. Since
-joining different databases is pretty common in MS SQL, this is an important
-issue. The question now is: How can Babelfish model the desired MS SQL behavior?
-In the case of `single-db`, MS SQL databases are mapped to schemas in PostgreSQL. In
-the case of `multi-db`, many databases are used instead of just one.
+When you create a Babelfish cluster, you choose between using a single
+SQL Server database on its own or multiple SQL Server databases together.
+Your choice affects how the names of SQL Server schemas inside the Babelfish
+database appear in PostgreSQL.
+
+If you specify `single-db`, you can create only a single SQL Server database in
+Babelfish, and SQL Server schemas will be created as regular PostgreSQL schemas
+in your Babelfish database.  If you specify `multi-db`, you can create
+several SQL Server databases with schemas in them, and an SQL Server schema will
+be created as PostgreSQL schema `<database name>_<schema_name>` to avoid name
+collisions.
+
+The migration mode is stored in the `migration_mode` parameter. You can't change
+this parameter after creating your cluster.
 
 During the deployment process you can decide which mode to use.
 How does this work? Consider the following code snippet:  
```

Comment:
change 'can' to 'must', and point out that the default is single-db.

Reply:
Ah, yes, that is missing. I have added that, used "must" and reorganized the paragraphs somewhat.